node/tests/ServerModules.test.ts

```diff
@@ -2679,7 +2751,54 @@ describe("Server Module Tests", () => {
                         condition: "used",
                         bicycles: isResp3 ? 4 : "4",
                     },
-                ]);
+                ];
+                expect(
+                    aggreg
+                        .map(convertGlideRecordToRecord)
+                        // elements (records in array) could be reordered
+                        .sort((a, b) =>
+                            a["condition"]! > b["condition"]! ? 1 : -1,
+                        ),
+                ).toEqual(expectedAggreg);
+
+                const aggregProfile: [
+                    FtAggregateReturnType,
+                    Record<string, number>,
+                ] = await GlideFt.profileAggregate(
```

Comment:
I think you can omit type there
```suggestion
                const aggregProfile = await GlideFt.profileAggregate(
```

Reply:
I prefer to add the type.